.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,13 @@
+name: Feature Request
+description: Request an enhancement
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        *Before requesting, search [existing issues](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) and our [changelog](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases) to see if the feature has already been addressed.*
```

I don't think the issues are always tagged appropriately. So, we should just link the whole issues list
```suggestion
        *Before requesting, search [existing issues](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/issues?q=is%3Aissue) and our [changelog](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases) to see if the feature has already been addressed.*
```